Assets/MRTK/SDK/Features/Utilities/Migration/Tools/MigrationTool.cs

```diff
@@ -362,6 +418,34 @@ private static bool IsSceneAsset(Object selectedObject)
         {
             return selectedObject is SceneAsset;
         }
+
+        /// <summary>
+        /// Utility class to keep migration status of each object
+        /// </summary>
+        public class MigrationStatus
+        {
+            /// <summary>
+            /// Flag to indicate if object was already processed by migration
+            /// </summary>
+            public bool isProcessed { get; set; }
```

Comment:
I think that for public properties we still want IsProcessed (same for Failures and Log below)